ui/components/multichain/pages/send/send.js

```diff
@@ -183,7 +191,8 @@ export const SendPage = () => {
   const onSubmit = async (event) => {
     event.preventDefault();
 
-    await dispatch(signTransaction());
+    setIsSubmitting(true);
+    await dispatch(signTransaction(history));
```

Comment:
Does the page fail gracefully if signing a swap + send transaction/approval fails? Example concerns: page getting stuck in loading state, user doesn't get feedback about the failed signing

Reply:
It did not – updated